app/controllers/users_controller.rb

```diff
@@ -8,12 +8,19 @@ def show
       conn = Faraday.new("https://api.github.com") do |req|
         req.headers["authorization"] = @user.token
       end
-      
-      repos = conn.get("/user/repos")
-      parsed = JSON.parse(repos.body, symbolize_names: true)
-      @repos = parsed.map do |repo_data|
-        Repo.new(repo_data)
-      end.first(5)
+
+      response = conn.get("/user/repos")
+      parsed = JSON.parse(response.body, symbolize_names: true)
+        @repos = parsed.map do |repo_data|
+          Repo.new(repo_data)
+        end.first(5)
+
+      response_followers = conn.get("/user/followers")
+      parsed_2 = JSON.parse(response_followers.body, symbolize_names: true)
+        @followers = parsed_2.map do |follower_data|
+          Follower.new(follower_data)
+        end.first(5)
+        # binding.pry
```

Comment:
put this in the same controller method and used the same faraday api call. This way it only works if it passes the `if` statement.

Reply:
Good call